Add a mapping helper for welding airlocks

Maurukas opened the pull request:
## About The Pull Request
What it says on the tin. Warning: copy-pasted mapper sprites.

## Why It's Good For The Game

Fewer varedits, better maintainability, more obvious in the map editor what's been applied to a given airlock.


## Changelog

No player facing changes.

Maurukas committed:
Adds mapping helper for welding airlocks

Jolly-66 commented:
Bless being able to weld airlocks instead of using wooden barricades 
Good PR

san7890 commented on code/modules/mapping/mapping_helpers.dm:
```diff
@@ -216,6 +216,16 @@
+	if(airlock.welded)
+		log_mapping("[src] at [AREACOORD(src)] tried to make [airlock] welded but it's already welded closed!")
+	else
+		airlock.welded = TRUE
```
does it show up as welded in-game? might have to do `update_appearance()` on the `airlock`

Maurukas replied:
It's in LateInitialize() on the parent. Also I tested it.

Maurukas committed:
Updath Paths + run

Curiously, there was only a single instance of manually welding an airlock in tgstation

Mothblocks commented on code/modules/mapping/mapping_helpers.dm:
```diff
@@ -216,6 +216,16 @@
+/obj/effect/mapping_helpers/airlock/welded/payload(obj/machinery/door/airlock/airlock)
+	if(airlock.welded)
+		log_mapping("[src] at [AREACOORD(src)] tried to make [airlock] welded but it's already welded closed!")
+	else
```
Should remove the else because it's unnecessary and makes this slightly harder to read

san7890 commented:
![image](https://user-images.githubusercontent.com/34697715/227615086-eabadb18-9ff9-4be5-8dd0-71aaca47325f.png)

CI is failing, that's my fault i forgot to mention that the updatepaths i fed you was just an example because i didn't bother to copy the real path

san7890 commented on _maps/RandomRuins/SpaceRuins/oldstation.dmm:
```diff
@@ -8768,9 +8768,9 @@
-	welded = 1
+	name = "Custodial Closet"
 	},
+/obj/effect/mapping_helper/welded,
```
```suggestion
/obj/effect/mapping_helpers/airlock/welded,
```

Maurukas committed:
i blame san entirely

fixies + mothblocks suggestions
no I'm not going to go and do every helper in this pr
